integrationtests/smb_test.go

```diff
@@ -7,7 +7,7 @@ import (
 	"math/rand"
 	"os"
 	"os/exec"
-        "strings"
```

Comment:
Since there are many test fixes in this, could we do it as separate PRs, so that we can merge one and work on the comments for another one ?

Reply:
I spitted it
https://github.com/kubernetes-csi/csi-proxy/pull/22